packages/arb-token-bridge-ui/src/types/ChainQueryParam.ts

```diff
@@ -0,0 +1,131 @@
+import { Chain } from 'wagmi'
+import * as chains from 'wagmi/chains'
+
+import {
+  ChainId,
+  getCustomChainsFromLocalStorage,
+  getSupportedNetworks
+} from '../util/networks'
+import * as customChains from '../util/wagmi/wagmiAdditionalNetworks'
+
+const chainQueryParams = [
+  'ethereum',
+  'goerli',
+  'sepolia',
+  'arbitrum-one',
+  'arbitrum-nova',
+  'arbitrum-goerli',
+  'arbitrum-sepolia',
+  'stylus-testnet',
+  'xai-testnet',
+  'local',
+  'arbitrum-local'
+] as const
+
+export type ChainQueryParam = (typeof chainQueryParams)[number]
```

Comment:
Maybe we should call this `ChainKeyQueryParam`? Meaning that `ethereum`, `arbitrum-one` etc. are of type `ChainKeyQueryParam` but `type ChainQueryParam = ChainKeyQueryParam | number` if we pass in id directly